Week-5/Assignment-4/src/calculator.test.js

```diff
@@ -0,0 +1,182 @@
+const mathOperations = require("./calculator");
+
+test("sum(2, 2) should be defined", () => {
+    expect(mathOperations.sum(2, 2)).toBeDefined();
+});
+
+test("sum(2, 2) should be 4", () => {
+    expect(mathOperations.sum(2, 2)).toBe(4);
+});
+
+test("sum(2, 2) should be greater than 3", () => {
```

Comment:
The intention of adding more test cases should reflect in choosing different set of arguments and not adding different assertion statements for same set of inputs.
What should happen let's say if i pass

1, undefined
1, null
undefined, undefined
1, "sds"
"sas", 1
"dsds", undefined

There are multiple scenarios like this. I expect you to cover those scenarios.

Reply:
Thanks @sairamin, I've updated my test cases now, please review